packages/api/src/external/commonwell/patient-conversion.ts

```diff
@@ -24,6 +24,19 @@ export function mapGenderAtBirthToCw(k: GenderAtBirth): GenderCodes | undefined
   return genderMapping[k];
 }
 
+export function mapGenderAtBirthFromCw(k: string): GenderAtBirth {
+  switch (k) {
+    case GenderCodes.F:
+      return "F";
+    case GenderCodes.M:
+      return "M";
+    case GenderCodes.UN:
+      return "O";
+    default:
+      return "U";
+  }
```

Comment:
ditto: avoid mapping those again

Reply:
after doing a sweep I actually dont think this function exists yet. 
I would rather just rethink the placement of this. 
we have fhir specific gender functions in `core/src/external/fhir/patient/index.ts` and cq specific functions in`core/src/external/carequality/shared.ts` 
Maybe this similarily belongs in commonwell/shared

Also naming convention. Lets continue using MetriportGender > GenderAtBirth and name the function like the others:

mapCwGenderToMetriportGender

Reply:
It doesnt get used anymore so removed

Reply:
re: naming convention

Agree, @jonahkaye. We should rename `mapGenderAtBirthToCw` to `mapMetriportGenderToCwGender`, to match `mapMetriportGenderToFhirGender` - not part of this PR, though. :)